templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,55 @@ contents:
       echo ""
     }
 
+    get_ip_from_ip_hint_file() {
+      local ip_hint_file="$1"
+      if [[ ! -f "${ip_hint_file}" ]]; then
+        return
+      fi
+      ip_hint=$(cat "${ip_hint_file}")
+      echo "${ip_hint}"
+    }
+
+    # This function waits for ip address of br-ex to be bindable
+    # This is workaround for OCPBUGS-673 as it will not allow starting crio
+    # before address is bindable
+    try_to_bind_address() {
+      # Retry for 5 minutes
+      retries=300
+      until [[ ${retries} -eq 0 ]]; do
+        ip=$(ip -j addr | jq -r ".[] | select(.ifname==\"br-ex\") | .addr_info[] | select(.scope==\"global\") | .local")
```

Comment:
This is failing in dual stack because the jq returns both addresses. I think we may need to select for ipv6 specifically, and maybe skip the bind entirely if there isn't one.

Reply:
Thanks a lot, will change it. Sounds reasonable